config/locales/to_change_everything/pt.yml

```diff
@@ -0,0 +1,305 @@
+      anarchy:
+        first_html: |-
+          <p><span class="green black">Anarquia</span> é o que acontece quando a ordem não é imposta pela força. É a liberdade: o processo de reinventar a nós e nossas relações continuamente.</p>
+                  <p>Qualquer processo que funciona organicamente – uma floresta tropical, um círculo de amizades, seu próprio corpo – é uma harmonia anárquica que persiste por meio de constante mudança. O controle de cima para baixo, por outro lado, só pode ser mantido pela contenção e pela coerção: a disciplina precária das salas de aula, as fazendas industriais onde pesticidas e herbicidas protegem fileiras estéreis de milho geneticamente modificados, a hegemonia frágil de uma superpotência.</p>
+        second_html: |-
+          <p><span class="pink black">Anarquismo</span> é a ideia de que todas as pessoas têm o direito à completa autodeterminação. Nenhuma lei, nenhum governo ou processo de decisão é mais importante que as necessidades e os desejos de seres humanos reais. As pessoas devem ser livres para moldarem suas relações de acordo com suas satisfações mútuas e defenderem a si mesmas quando se sentirem aptas a isso.</p>
+                    <p>Anarquismo não é um dogma ou um mapa. Não é um sistema que supostamente funcionaria bastando que fosse aplicado corretamente, como a democracia, nem é um objetivo para ser realizado em um futuro muito distante, como o comunismo. É uma forma de agir e nos relacionar que podemos pôr em prática imediatamente. Ele oferece questionamentos que podemos colocar em qualquer situação. Ao invés de questionar se a forma como se desenrola uma ação é legal, ou violenta, ou bondosa, podemos perguntar simplesmente: como ela distribui o poder?</p>
+        third_html: |-
+          <p><span class="blue black">Anarquistas</span> se opõem a toda forma de hierarquia – toda prática que concentra poder na mão de poucas pessoas, todo mecanismo que nos põe distantes de nosso potencial. Contra os sistemas fechados, apreciamos o desconhecido frente a nós, o caos interior através do qual somos capazes de ser livres.</p>
+                    <img src="https://cloudfront.crimethinc.com/tce/images/circle-a-logo-hands-white-large@2x.png">
+      outro:
+        left_text_html: "<p>Quando vemos o que todas as diferentes instituições
+          e os mecanismos de dominação têm em comum, torna-se claro que nossas
+          lutas individuais são também parte de algo maior que nós, algo que pode
+          nos conectar. Quando nos unimos em torno dessa conexão, tudo muda: não
+          apenas nossas lutas, mas também nosso senso de potencial de agir, nossa
+          capacidade de desfrutar, a sensação de que nossas vidas têm sentido.
+          Tudo o que precisamos para nos encontrar umas com as outras é começar
+          a agir de acordo com uma lógica diferente.</p>"
+        right_text_html: |-
+          <h3>Para <br>
```

Comment:
i think this is a bug in my scraper script.

This `<h3>`  shouldn't be here as it already gets wrapped in an h3 in [`_spread.html.erb`][0]]

```erb
  <div class="rightintro">
    <h3><%= t "to_change_everything.show.#{section}.right_text_html" %></h3>
  </div>
```

[0]:https://github.com/crimethinc/website/blob/master/app/views/to_change_everything/_spread.html.erb#L9